src/index.js

```diff
@@ -1,2 +1,40 @@
-import _ from "lodash";
-import "./index.css";
+import './index.css';
+
+const playerNmae = document.querySelector('.player-name');
+const playerScore = document.querySelector('.player-score');
+const formEle = document.getElementById('form-ele');
+const refreshScore = document.getElementById('refresh');
+
+const gameId = '2tfc8KffSa4kon6LcmTZ';
+const gameUrl = `https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${gameId}/scores`;
+
+formEle.addEventListener('submit', async (e) => {
+  e.preventDefault();
+  const formData = {
+    user: playerNmae.value,
+    score: playerScore.value,
+  };
+  const checkIn = await fetch(gameUrl, {
+    method: 'POST',
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+    body: JSON.stringify(formData),
+  });
+  if (!checkIn.ok) {
+    return;
+  }
+  await checkIn.json();
+  e.target.reset();
+});
+
+refreshScore.addEventListener('click', async () => {
+  const res = await fetch(gameUrl);
+
+  const data = await res.json();
+  let tableData = '';
+  data.result.forEach((values) => {
+    tableData += `<li> ${values.user}: ${values.score}</li>`;
+  });
+  document.getElementById('table-content').innerHTML = tableData;
+});
```

Comment:
When loading the application the first time or after reload, the data should be fetched automatically and populated into the UI. However, in your case, it doesn't. The data is only populated when the user hits the refresh button. The refresh button is to be after submitting a new score and to display that score on UI. Kindly fix this issue.